src/openapi/streaming/serializer/serializer-protobuf.js

```diff
@@ -0,0 +1,153 @@
+import MetaProcessor from './meta/meta-protobuf';
+import wrappers from './wrappers/protobuf-wrappers';
+import protobuf from 'protobufjs/dist/protobuf.min';

```

Comment:
It produces smaller openapi-client lib size, but in the end, when minified it should produce same size as with this solution. So it's less scary in non-minified version. Other then that no advantages. 

As for using webpack instead of rollup. Not sure if it should be done as part of this PR. But in the end, as Webpack doesn't support tree shaking, I doubt it would preduce better results.

Reply:
As discussed, let try to implement passing of protobuf from openapi-clientlib userspace.